src/main/java/de/tum/in/www1/artemis/repository/ComplaintRepository.java

```diff
@@ -50,8 +51,8 @@
      * @param courseId  the id of the course
      * @return the number of unaccepted
      */
-    @Query("SELECT count(c) FROM Complaint c  WHERE c.student.id = :#{#studentId} AND c.result.participation.exercise.course.id = :#{#courseId} AND (c.accepted = false OR c.accepted is null)")
-    long countUnacceptedComplaintsByStudentIdAndCourseId(@Param("studentId") Long studentId, @Param("courseId") Long courseId);
+    @Query("SELECT count(c) FROM Complaint c  WHERE c.complaintType <> :#{#complaintType} AND c.student.id = :#{#studentId} AND c.result.participation.exercise.course.id = :#{#courseId} AND (c.accepted = false OR c.accepted is null)")
+    long countUnacceptedComplaintsByStudentIdAndCourseId(@Param("studentId") Long studentId, @Param("courseId") Long courseId, @Param("complaintType") ComplaintType complaintType);
```

Comment:
I suggest to change the name of the method, because you don't understand from the name alone why there is a `complaintType` parameter

Reply:
@rpadovani As I saw a part of the implementation commented out it was in the complaints component and was a part of it. Of course, it can be as a more sophisticated feature. But for now tutors are also not allowed to change the grade for after student requested more feedback